### PR TITLE
[KafkaHub] Update consumer secure-socket configuration retrieval logic

### PR DESCRIPTION
**Pull request by ayeshLK**
## Examples
N/A

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec
- [ ] Checked native-image compatibility

### CODE REVIEW

File: examples/kafka-hub/hub/Config.toml [hunk 1]
```diff
@@ -48,5 +48,11 @@ cert.password = "password"
 key.keyStore.path = "./resources/brokercerts/client-keystore.jks"
 key.keyStore.password = "password"
 
+# The client MTLS configurations used by Kafka consumers mapped to HTTP subscribers
+[kafkaHub.config.KAFKA_CLIENT_KS_CONFIGS."test-consumer-group"]
+consumerGroupName="test-consumer-group"
+key.keyStore.path = "./resources/brokercerts/test-consumer-group-client-keystore.jks"
+key.keyStore.password = "password"
```
Comment (shafreenAnfar) [inline examples/kafka-hub/hub/Config.toml:55]: We need to be able to give both keystore and turststore

Comment (ayeshLK) [reply examples/kafka-hub/hub/Config.toml:55]: Why do we need the truststore ? They are connecting to the same kafka-cluster right ?

File: examples/kafka-hub/hub/modules/config/configurations.bal [hunk 2]
```diff
@@ -68,3 +68,6 @@ public final string WEBSUB_EVENTS_CONSUMER_GROUP = os:getEnv("WEBSUB_EVENTS_CONS
 isolated function constructSystemConsumerGroup() returns string {
     return string `websub-events-receiver-${SERVER_IDENTIFIER}-${util:generateRandomString()}`;
 }
+
+# The client MTLS configurations used by Kafka consumers mapped to HTTP subscribers
+public configurable map<types:KafkaClientKeyStoreConfig> KAFKA_CLIENT_KS_CONFIGS = ?;
```
Comment (NipunaRanasinghe) [inline examples/kafka-hub/hub/modules/config/configurations.bal:73]: We usually dont use snake case unless its a compile-time constant right? shouldn't we stick to camel case in here?
```suggestion
public configurable map<types:KafkaClientKeyStoreConfig> kafkaClientKSConfigs = ?;
```

File: examples/kafka-hub/hub/modules/types/types.bal [hunk 3]
```diff
@@ -38,3 +38,11 @@ public type KafkaMtlsConfig record {|
         string keyPassword?;
     |}|kafka:CertKey key?;
 |};
+
+public type KafkaClientKeyStoreConfig record {|
+    string consumerGroupName;
+        record {|
```
Comment (NipunaRanasinghe) [inline examples/kafka-hub/hub/modules/types/types.bal:44]:
```suggestion
    record {|
```

**Pull request closed without merging**